homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -1,13 +1,15 @@
 """Support for Frontier Silicon Devices (Medion, Hama, Auna,...)."""
 import logging
 
-from afsapi import AFSAPI
+# use local override until https://github.com/zhelev/python-afsapi/pull/4 is accepted
+from .afsapi import AFSAPI
```

Comment:
This should not be added to Home Assistant during the transition period. Instead, this PR should be opened when it becomes available.

Reply:
I fully agree and I really would have done it that way.
However, given the low activity on the upstream library (last commit is three years ago, same for last activity from the author on GitHub) I'm afraid this will take a long time - if it ever gets merged at all.
I like the feature though (and know other people would appreciate), that's why I still opened the PR and added the code in a way that it can easily be switched to upstream should my fears have been unjustified. In that case I would raise the new PR of course.
Do you think this one could be acceptable that way?

Reply:
> Do you think this one could be acceptable that way?

I cannot accept this PR in its current state, hence my review comment.

Reply:
Ok, removed the "local override" part. Is it better that way?